Sources/Helpers/OASavingTrackHelper.h

```diff
@@ -12,9 +12,17 @@
 #define kTrackNoHeading -1.0
 
 @class OAGPXMutableDocument;
+@class OAGPXDocument;
 @class OAGPX;
 @class OAWptPt;
 
+@protocol OATrackSavingHelperUpdatableDelegate <NSObject>
```

Comment:
1 - `OASavingTrackHelper` is backend class and should not have any UI code.
2 - It is not clear why added method are needed. They should be already present in current codebase. (Looks like need to be discussed)